File: The Core/Intro Gates/6. Circle Of Numbers/circleOfNumbers.py
def circleOfNumbers(n, firstNumber):
    nums = []
    
    for x in range(n):
        nums.append(x)
    
    if firstNumber < (n/2):
        return nums[int(firstNumber + (n/2))]
    else:
        return nums[int(firstNumber - (n/2))]

File: The Core/Intro Gates/6. Circle Of Numbers/circleOfNumbers.lua
function circleOfNumbers(n, firstNumber)
    nums = {}
    
    for i = 0, n do
        table.insert(nums, i)
    end
    
    if firstNumber < (n/2) then
        return nums[firstNumber + (n/2) + 1]
    else
        return nums[firstNumber - (n/2) + 1]
    end    
end
